assets/airlines/kor.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Air Koryo",
+    "callsign": {
+        "name": "Air Koryo",
+        "length": 3
+    },
+    "fleets": {
+        "default": [
+            ["A148",    2],
```

Comment:
this aircraft doesn't exist in this PR, so it causes an error when trying to spawn one. It may be included in PR #593, but it is missing from this one.